probe/endpoint/reporter.go

```diff
@@ -62,7 +62,7 @@ func NewReporter(hostID, hostName string, includeProcesses bool, useConntrack bo
 		}
 	}
 	if conntrackModulePresent {
-		natmapper, err = newNATMapper()
+		natmapper, err = NewNATMapper()
```

Comment:
I agree - but the test need to be in a separate package in this instance, as testing imports endpoint, so endpoint can't input testing.  In general though I much prefer tests in the same package as the code for this very reason.